src/components/v5/common/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+import debounce from 'lodash.debounce';
+
+import FormError from '~v5/shared/FormError';
+
```

Comment:
No necessary blank line between component import and types.
I think the good way is to divide imports into two groups:
- on the top: import packages like react and other plugins, etc.
- below: imports of components, hooks, utils, types and everything what we created by ourselves